apps/maps/calitp_map_utils/__init__.py

```diff
@@ -0,0 +1,167 @@
+import base64
+import gzip
+import json
+import os
+from enum import Enum
+from typing import Any, Dict, Optional, Tuple, Union
+
+import requests
+import typer
+from calitp_data.storage import get_fs  # type: ignore
+from furl import furl
+from geojson_pydantic import Feature, FeatureCollection, MultiPolygon, Point, Polygon
+from geojson_pydantic.geometries import Geometry
+from geojson_pydantic.types import Position
+from pydantic import BaseModel, Field, HttpUrl, ValidationError, conlist, root_validator
+from tqdm import tqdm
+
+MAP_APP_URL_ENV_VAR = "CALITP_MAP_APP_URL"
+MAP_APP_URL = os.getenv(MAP_APP_URL_ENV_VAR)
+
+
+class LayerType(str, Enum):  # name?
```

Comment:
LayerType is optional and it's primarily for coordinating styling work that's done in the Javascript with a common analysis with known needs. Most of the actual map attributes are passable via the state or feature properties, but @edasmalchi suggested trying to avoid requiring data in properties so that users are still downloading largely vanilla GeoJSON.

I think a general workflow could be:
1. New data layer can provide styling in properties for testing without being blocked on frontend changes
2. Create a new LayerType and define required data properties (e.g. `avg_mph`)
3. Add styling in the Javascript based on those properties, so the GeoJSON does not need the actual color/style props